packages/docusaurus/docs/getting-started/migrating/benefits.md

```diff
@@ -16,4 +16,4 @@ While the Yarn Classic line (1.x) remains a pillar of the JavaScript ecosystem,
 
 - **Extensibility:** Yarn Modern's architecture allows you to build your own features as you need it. No need to wait for us to implement this feature you hope for - you can now implement it yourself, according to your own specs, and use it straight away! Focused workspaces, custom installs, project validation, ...
 
-- **Future proof:** Yarn Modern was built after we starting to see how difficult it was to build new features on Yarn Classic, with most changes having unpredictable consequences. This plateau has been solved, as evidenced by the slate of features that we released in the past few major releases.
+- **Futureproofing:** Yarn Modern was built when we saw how difficult it was to build new features on Yarn Classic, with most changes having unpredictable consequences. This plateau has been solved in Yarn Modern, as we have proven with the dozens of new features in the past few releases.
```

Comment:
I understand we're in the middle of Hacktoberfest shitshow and I disagree with most of the changes here but "after we starting to see" sounds like a genuine thing to fix